draft-ietf-idr-deprecate-as-set-confed-set.xml

```diff
@@ -307,11 +287,17 @@ Route aggregation that was previously performed by proxy aggregation without the
         <t>In order to ensure a consistent BGP origin AS is announced for
         aggregate BGP routes for implementations of "brief" BGP aggregation, the
         implementation should be configured to truncate the AS_PATH after the
-        right-most instance of the desired origin AS for the aggregate.</t>
+        right-most instance of the desired origin AS for the aggregate. The desired origin AS could be the aggregator AS itself.</t>
 
         <t>If the resulting AS_PATH would be truncated from the otherwise
-        expected result of BGP AS_PATH aggregation (an AS_SET would be
-        generated, or ASes are removed from the "longest leading sequence" of
+        expected result of BGP AS_PATH aggregation (an AS_SET would not be
+        generated, and/or ASes are removed from the "longest leading sequence" of
+
+<!-- 

```

Comment:
The change above seems right.  Remove comment.

Reply:
done